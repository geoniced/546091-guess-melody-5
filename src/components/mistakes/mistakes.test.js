import React from "react";
import renderer from "react-test-renderer";
import Mistakes from "./mistakes";

describe(`Mistakes render correctly`, () => {
  it(`Renders count equals zero`, () => {
    const tree = renderer
      .create(
          <Mistakes
            count={0}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Renders count equals one`, () => {
    const tree = renderer
      .create(
          <Mistakes
            count={1}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

