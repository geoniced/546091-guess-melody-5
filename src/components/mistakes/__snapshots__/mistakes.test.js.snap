// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes render correctly Renders count equals one 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;

exports[`Mistakes render correctly Renders count equals zero 1`] = `
<div
  className="game__mistakes"
/>
`;
